data_pump/eperson.py

```diff
@@ -62,6 +67,22 @@ def import_eperson(metadata_class,
     logging.info("Eperson was successfully imported!")
 
 
+'''
+The eperson email could consist of more email, return all of them in the array  
+'''
+
+
+def get_eperson_emails(email):
+    if email is None:
+        return []
+
+    if ';' not in email:
+        return [email]
+
+    # email value contains of two email, take just the first one.
+    # e.g., test@msn.com;name@gmail.com
+    return email.split(';')
```

Comment:
and why if it contains only from one mail

Reply:
It is guaranteed by the condition above:
```
    if ';' not in email:
        return [email]
```